internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -227,6 +235,13 @@ func resourceLogAnalyticsWorkspaceCreateUpdate(d *pluginsdk.ResourceData, meta i
 		sku.Name = workspaces.WorkspaceSkuNameEnumPerGBTwoZeroOneEight
 	}
 
+	// convert features to interface{} to use 2020-08-01 version API, the 2022-10-01 version API has strict limitation on `reservation_capacity_in_gb_per_day` which will cause breaking change.
```

Comment:
Is there more info on what breaking changes there are If we can't update the API version here?